packages/api-server/api_server/routes/delivery_alerts.py

```diff
@@ -35,6 +40,8 @@ async def respond_to_delivery_alert(
         task_id=task_id,
         message=message,
     )
+    logger.info(f"Delivery alert responded by {user.username}")
+    logger.info(delivery_alert)
```

Comment:
In our case it doesn't matter since we only run a single thread (for now), but in general it is better either
1. Put both into a single log so it can be easily identified.
2. Include the request id so it can be traced, that is if we/fastapi have a system to uniquely tag each request.

Reply:
The task ID is included in the delivery alert model